_posts/2021-04-17-dd_00001_intro.md

```diff
@@ -0,0 +1,324 @@
+transistor anymore and could put more time and effort into the actual design of
+their system. Two HDLs emerged in the 1980s: VHDL and Verilog. Those are still
+the main languages to use when it comes to digital design. In the last couple of
+decades, new trends in digital design emerged, among which the most popular was
+**High-Level Synthesis** (HLS). The idea of HLS is to take an algorithm written
+in some high-level programming language, such as C, and automatically generate
+an optimized digital circuit for that particular algorithm. To this date,
+researchers all over the world are trying to come up with new languages and
+design paradigms with the hope of eventually replacing the traditional HDLs.
+
+The rapid advancement of microchips over the past decades was - apart from the
+technological improvements in transistors - made possible by EDA. EDA tools
+automate many time-consuming tasks in chip design. Two of the most important
+tasks are _Synthesis_ and _Place & Route_. In Synthesis, the HDL code is
+translated into a logic gates circuit (not a physical one, but described in some
+file). Then, in Place&Route, the EDA tool tries to find the optimal location for
+each gate on the final chip. Both tasks, try to follow certain _constraints_
+given by the engineer, such as the final performance or power consumption of the
+chip. The tools use physical models of the actual gates to come up with
+estimations on performance or power consumption. The engineer can then analyze
+the results, and refine his design if he is not satisfied.
+
+Another important aspect during the design of a digital circuit is **testing**.
+Before the chip is manufactured, we have to make sure that everything is working
+as intended. Consider this: When you ship _software_ to a customer, and he/she
+finds a bug, you can easily patch it through a _software update_. However, what
+do you do when a customer finds a bug in your chip? How do we fix bugs in chips?
+The answer is: we don't, because there is no such thing as a "hardware update"
+(some chips actually can be _reconfigured_, but we will cover them in a later
+post). Even if we could, doing this for potentially millions of sold chips, is
+straight-up infeasible. That's why a LOT of time is put into testing before
+production. This is especially important for safety-critical applications such
+as planes or medical devices.
+
+There are several ways to test a digital circuit. A commonly used method is
+**simulation**. In simulation, our HDL design files are read and test signals
+are applied to see whether the design behaves as intended. If an error is found
+during simulation, the designer analyzes the circuit and fixes the bug.
+Simulation is just one part of a whole sub-branch in digital design:
+**Verification**. Over the decades, with the rising complexity of chips, a lot
+of effort has been put into coming up with efficient verification methods.
```

Comment:
Do you want to mention that it is not possible to apply all stimuli because of the complexity?

Reply:
I think this would be already too detailed for a newbie?

Reply:
I also mentioned that due to the complexity, new verification methods are applied as well.